.github/pull_request_template.md

```
@@ -0,0 +1,27 @@
+- Performance improvements must demonstrate nontrivial, measurable gains
+  (include before/after numbers).
+- Refactors, style changes, and "improvements" without a concrete
+  problem being solved are likely to be closed.
+- Js conversion to ts are in general welcome, but if the queue is long 
+  they will often go stale before getting attention.
+
+If you cannot articulate what real problem this solves for users,
+it probably should not be a PR at this time. 
+-->
+
+## What problem does this solve?
```

markdownlint-cli2 (0.22.1)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)